sklearn/utils/tests/test_validation.py

```diff
@@ -1213,3 +1213,71 @@ def test_check_sparse_pandas_sp_format(sp_format):
     assert sp.issparse(result)
     assert result.format == sp_format
     assert_allclose_dense_sparse(sp_mat, result)
+
+
+def new_pandas_version():
+    pd = pytest.importorskip("pandas")
+    modv = getattr(pd, '_version')
+    dct = modv.get_versions()
+    ver = dct['version']
+    vnums = [int(n) for n in ver.split('.')]
+    is_new = vnums[0] > 1 or (vnums[0] == 1 and vnums[1] > 0)
+    return is_new
+
+
+@pytest.mark.skipif(new_pandas_version(),
+                    reason="issue of generating an object dtype "
+                    "coo_matrix from a DataFrame with extension "
+                    "arrays with different numeric types was fixed"
+                    " in pandas 1.1.0")
+@pytest.mark.parametrize('ntype1, ntype2', [
+    ("longdouble", "float16"),
+    ("float16", "float32"),
+    ("float32", "double"),
+    ("int16", "int32"),
+    ("int32", "long"),
+    ("byte", "uint16"),
+    ("ushort", " uint32"),
+    ("uint32,", "uint64"),
+    ("uint8", "int8"),
+])
+def test_check_pandas_sparse_invalid(ntype1, ntype2):
+    pd = pytest.importorskip("pandas")
```

Comment:
We can skip here instead when `pandas` is above 1.1.0:

```py
pd = pytest.importorskip("pandas")
if parse_version(pd.__version__) >= parse_version('1.1'):
	pytest.skip(...)
```

Reply:
done